R/fetchComponents.R

```diff
@@ -15,29 +15,108 @@ fetchComponents <- function(sys) {
   # returns a fixed set of symbols.
   # Function stub for development purposes only.
 
-  message("Note: fetchComponents(...) is deprecated.",
```

Comment:
There are some changes within Dr. Steipe's functions

Reply:
Hey, I can't find an inconsistency between the two files save for formatting.

Reply:
... and that `fetchComponents()` produces the "deprecated" warning. But if possible, switch to `SydDBgetSysSymbols()`.

Reply:
Ideally, just pull the updated version, then the file should not even appear here.